tap_s3_csv/s3.py

```diff
@@ -225,7 +232,14 @@ def list_files_in_bucket(bucket, search_prefix=None):
 @retry_pattern()
 def get_file_handle(config, s3_path):
     bucket = config['bucket']
-    s3_client = boto3.resource('s3')
+    aws_endpoint_url = config['aws_endpoint_url']
```

Comment:
it gives `KeyError: 'aws_endpoint_url'` when it's missing from `config.json`

Reply:
adjusted to use config.get('aws_endpoint_url'), which should return None, if it doesn't exist.